backend/app/views/spree/admin/products/_form.html.erb

```diff
@@ -194,7 +194,14 @@
       <%= f.label :taxon_ids, Spree.t(:taxons) %>
 
       <% if can? :modify, Spree::Classification %>
-        <%= f.select :taxon_ids, options_from_collection_for_select(@product.taxons, :id, :pretty_name, @product.taxon_ids), { include_hidden: true }, multiple: true, class: 'select2-hidden-accessible' %>
+        <%= f.select :taxon_ids, options_from_collection_for_select(@product.taxons, :id, :pretty_name, @product.taxon_ids), 
```

Comment:
@damianlegawiec Because The old javascript picks up off the ID here, I was thinking it might be best to change the ID from `product_taxon_ids` to `ProductTaxonIds`

This way we are not running the API V1 and API V2 fetch request together by mistake, and we can ensure that any good results we are getting, are actually coming from the changes we are making, are you okay with this?